Babylon/groups/azure/groups/acr/registry_connect.py

```diff
@@ -0,0 +1,34 @@
+import json
+import logging
+import subprocess
+
+from azure.containerregistry import ContainerRegistryClient
+from azure.identity import DefaultAzureCredential
+import docker
+
+logger = logging.getLogger("Babylon")
+
+
+def registry_connect(registry: str,
+                     credentials: DefaultAzureCredential) -> tuple[ContainerRegistryClient, docker.DockerClient]:
+    # Login to registry
+    registry_client = ContainerRegistryClient(f"https://{registry}",
+                                              credentials,
+                                              audience="https://management.azure.com")
+    registry_part = registry.split(".")[0]
+    response = subprocess.run(["az", "acr", "login", "--name", registry_part, "--expose"],
+                              shell=False,
+                              check=False,
+                              stdout=subprocess.PIPE,
+                              stderr=subprocess.PIPE)
+    decoded = json.loads(response.stdout)
+    # Login to docker with the returned access token
+    client = docker.from_env()
+    try:
+        client.login(username="00000000-0000-0000-0000-000000000000",
```

Comment:
username=resgistry also

Reply:
In fact username is literally 000, it does not work with username registry

Reply:
Ah Ok  😅 !